### PR TITLE
docs: --privileged docs completeness, consistency

### PR DESCRIPTION
**Pull request by markstos**
As discussed in https://github.com/containers/libpod/issues/4840

### CODE REVIEW

File: docs/source/markdown/podman-exec.1.md [hunk 1]
```diff
@@ -43,7 +43,21 @@ Pass down to the process N additional file descriptors (in addition to 0, 1, 2).
 
 **--privileged**
 
-Give the process extended Linux capabilities when running the command in container.
+Give extended privileges to this container. The default is *false*.
+
+By default, Podman containers are
+“unprivileged” and cannot, for example, modify parts of the kernel.
```
Comment (rhatdan) [inline docs/source/markdown/podman-exec.1.md]: change to modify parts of the OS.

File: docs/source/markdown/podman-exec.1.md [hunk 2]
```diff
@@ -43,7 +43,21 @@ Pass down to the process N additional file descriptors (in addition to 0, 1, 2).
...
+This is because by default a container is not allowed to access any devices.
```
Comment (rhatdan) [inline docs/source/markdown/podman-exec.1.md]: By default the container is only allowed limited access to devices.

File: docs/source/markdown/podman-exec.1.md [hunk 3]
```diff
@@ -43,7 +43,21 @@ Pass down to the process N additional file descriptors (in addition to 0, 1, 2).
...
+A “privileged” container is given access to all devices that the user creating
```
Comment (rhatdan) [inline docs/source/markdown/podman-exec.1.md]: A "privileged" container is given the same access to devices as the user launching the container.

File: docs/source/markdown/podman-exec.1.md [hunk 4]
```diff
@@ -43,7 +43,21 @@ Pass down to the process N additional file descriptors (in addition to 0, 1, 2).
...
+the container has permission to access.
+
+When the operator executes a privileged container, Podman enables access
```
Comment (rhatdan) [inline docs/source/markdown/podman-exec.1.md]: A privileged container, turns off the security features that isolate the container from the host.  Dropped
Capabilities, limited devices, read/only mount points, Apparmor/SELinux separation, and Seccomp filters are all
disabled.

File: docs/source/markdown/podman-exec.1.md (hunk 1 above)
Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman-exec.1.md]: OS to operating system

**Comment by TomSweeneyRedHat** [comment]
One small tweak to @rhatdan's wording, otherwise LGTM once refactored. TYVM @markstos !

**Comment by markstos** [comment]
Requested changed were made in an amended commit.

**Pull request merged**